admin-dev/themes/new-theme/js/pages/product/components/generator/CombinationGenerator.vue

```diff
@@ -77,29 +77,39 @@
   </div>
 </template>
 
-<script>
+<script lang="ts">
   import CombinationsService from '@pages/product/services/combinations-service';
-  import AttributesSelector from '@pages/product/components/generator/AttributesSelector';
+  import AttributesSelector, {Attribute, AttributeGroup}
+    from '@pages/product/components/generator/AttributesSelector.vue';
   import isSelected from '@pages/product/mixins/is-attribute-selected';
```

Comment:
Can we still call this a mixin? Since it's actually an extended Vue object?

Reply:
I don't think so, because a mixin is an object, not an extended vue object, right?